packages/node_modules/@ciscospark/redux-module-teams/src/index.js

```diff
@@ -0,0 +1,66 @@
+import {fromJS} from 'immutable';
+
+export const STORE_TEAMS = `user/STORE_TEAMS`;
+export const UPDATE_STATUS = `user/UPDATE_STATUS`;
+
+export const initialState = fromJS({
+  items: {},
+  status: {
+    isFetching: false,
+    hasFetched: false
+  }
+});
+
+
+export default function reducer(state = initialState, action) {
+  switch (action.type) {
+  case STORE_TEAMS:
+    return state.setIn([`status`, `hasFetched`], true)
```

Comment:
May want to set `isFetching` to false here as well.

Reply:
ooo yea, good catch.